project/Dependencies.scala

```diff
@@ -13,20 +13,14 @@
  * limitations under the License.
  */
 
-import sbt._
+import sbt.*
 
 object Dependencies {
 
   object Versions {
-    val spark2 = "2.4.8"
     val spark3 = "3.3.2"
 
     val scala211 = "2.11.12"
```

Comment:
I guess that this is still todo? Don't we want to drop 2.11 completely?

Reply:
In readme there's a description how to build for Spark 2.4, having this constant here make the switch easy.